src/ServiceBus/ServiceBus/ServiceBus.format.ps1xml

```diff
@@ -19,9 +19,16 @@
               <ListItem>
```

Comment:
@v-Ajnava as discussed offline, we should remove this entry for `ResourceGroup` and update the examples accordingly

Reply:
Resolved